broker/src/test/java/io/camunda/zeebe/broker/system/partitions/impl/StateControllerImplTest.java

```diff
@@ -84,6 +92,19 @@ public void shouldNotTakeSnapshotIfDbIsClosed() {
         .hasCauseInstanceOf(StateClosedException.class);
   }
 
+  @Test
+  public void shouldNotTakeSnapshotIfNoIndexedEntry() {
+    // given
+    final var snapshotPosition = 1;
+    exporterPosition.set(snapshotPosition - 1);
```

Comment:
:wrench: These two aren't needed apparently. Not sure if it's worth to remove but it threw me off a bit while reading the test because I didn't understand how they are related.